pkg/agent/const.go

```diff
@@ -60,14 +60,22 @@ const (
 )
 
 const (
-	// Windows custom scripts
-	kubernetesWindowsAgentCustomDataPS1   = "windows/kuberneteswindowssetup.ps1"
-	kubernetesWindowsAgentFunctionsPS1    = "windows/kuberneteswindowsfunctions.ps1"
-	kubernetesWindowsConfigFunctionsPS1   = "windows/windowsconfigfunc.ps1"
-	kubernetesWindowsKubeletFunctionsPS1  = "windows/windowskubeletfunc.ps1"
-	kubernetesWindowsCniFunctionsPS1      = "windows/windowscnifunc.ps1"
-	kubernetesWindowsAzureCniFunctionsPS1 = "windows/windowsazurecnifunc.ps1"
-	kubernetesWindowsOpenSSHFunctionPS1   = "windows/windowsinstallopensshfunc.ps1"
+	// kubernetesWindowsAgentCSECommandPS1 privides the command of Windows CSE
+	kubernetesWindowsAgentCSECommandPS1           = "windows/csecmd.ps1"
+	// kubernetesWindowsAgentCustomDataPS1 is used for generating the customdata of Windows VM
+	kubernetesWindowsAgentCustomDataPS1           = "windows/kuberneteswindowssetup.ps1"
```

Comment:
Currently we stores these files in `C:\AzureData\k8s` in Windows node. We need to assure that changing it from `k8s` to `windows` should not block anything including TSG
```
    Directory: C:\AzureData\k8s


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         1/1/1980  12:00 AM           2036 containerdtemplate.toml
-a----         1/1/1980  12:00 AM          11095 kuberneteswindowsfunctions.ps1
-a----         1/1/1980  12:00 AM          14533 windowsazurecnifunc.ps1
-a----         1/1/1980  12:00 AM           1151 windowscnifunc.ps1
-a----         1/1/1980  12:00 AM           8077 windowsconfigfunc.ps1
-a----         1/1/1980  12:00 AM           6129 windowscontainerdfunc.ps1
-a----         1/1/1980  12:00 AM           1762 windowscsiproxyfunc.ps1
-a----         1/1/1980  12:00 AM           1698 windowshostsconfigagentfunc.ps1
-a----         1/1/1980  12:00 AM           1816 windowsinstallopensshfunc.ps1
-a----         1/1/1980  12:00 AM          13970 windowskubeletfunc.ps1
```

Reply:
I was not able to find TSG related to these script.
Tested the case when k8s dir was used, and upgrading to windows did not break windows functions.

Reply:
@marinred I found that the path change here does not change the location in Windows nodes. These files are still stored in C:\AzureData\Windows\

Reply:
It should change the dir. I have a launched AKS cluster, and `k8s` dir instead of `windows` locates under C:/AzureData